File: src/main/java/com/polykhel/billnext/config/ApplicationProperties.java
package com.polykhel.billnext.config;

import org.springframework.boot.context.properties.ConfigurationProperties;

/**
 * Properties specific to Billnext.
 * <p>
 * Properties are configured in the {@code application.yml} file.
 * See {@link tech.jhipster.config.JHipsterProperties} for a good example.
 */
@ConfigurationProperties(prefix = "application", ignoreUnknownFields = false)
public class ApplicationProperties {}

File: src/main/kotlin/com/polykhel/billnext/config/ApplicationProperties.kt
package com.polykhel.billnext.config

import org.springframework.boot.context.properties.ConfigurationProperties

/**
 * Properties specific to BillNext.
 *
 * Properties are configured in the `application.yml` file.
 * See [tech.jhipster.config.JHipsterProperties] for a good example.
 */
@ConfigurationProperties(prefix = "application", ignoreUnknownFields = false)
class ApplicationProperties
